library/libraries.js

```diff
@@ -1068,5 +1068,15 @@ var d41d8cd98f00b204e9800998ecf8427e_LibraryDetectorTests = {
             }
             return false;
         }
+    },
+    'Moment.js': {
+        icon: 'momentjs',
+        url: 'http://momentjs.com/',
+        test: function(win) {
+            if(win.moment && win.moment.format) {
```

Comment:
`moment.format` is not a valid method.  Could you please change this to use something like `moment.isMoment`?